tests/test_photometryModel.py

```diff
@@ -170,7 +170,7 @@ def setUp(self):
         self.visitOrder = 3
         self.focalPlaneBBox = self.camera.getFpBBox()
         # tweak to get more than just a constant field for the second ccdImage
-        self.delta = np.arange(20, dtype=float)*-0.2 + 1
+        self.delta = (np.arange(20, dtype=float)*-0.2 + 1)[::-1]
```

Comment:
Why are you reversing the order of `delta` (if that's what this syntax is)?

Reply:
Added a comment: it makes the lowest order terms largest (which is what a realistic model would have).